includes/HealthChecks.php

```diff
@@ -0,0 +1,332 @@
+					'title'   => __( 'Object Caching', 'newfold-performance-module' ),
+					'pass'    => __( 'Object caching is enabled', 'newfold-performance-module' ),
+					'fail'    => __( 'Object caching is disabled', 'newfold-performance-module' ),
+					'text'    => __( 'Object caching saves results from frequent database queries, reducing load times by avoiding repetitive query processing. Object caching is available in all tiers of Bluehost Cloud.', 'newfold-performance-module' ),
+					'actions' => array(
+						array(
+							'label' => __( 'Learn more about Bluehost Cloud Hosting.', 'newfold-performance-module' ),
+							'url'   => 'https://www.bluehost.com/hosting/cloud',
+							'external' => true,
+						),
+					),
+					'test'    => function () {
+						return wp_using_ext_object_cache();
+					},
+				)
+			);
+		}
+
+		// PRESS7-107: Cloudflare.
+		$manager->addHealthCheck(
+			array(
+				'id'    => 'cloudflare-active',
+				'title' => __( 'Cloudflare enabled', 'newfold-performance-module' ),
+				'pass'  => __( 'Cloudflare integration is enabled', 'newfold-performance-module' ),
+				'fail'  => __( 'Cloudflare integration is disabled', 'newfold-performance-module' ),
+				'text'  => __( 'Cloudflare integration can improve performance and security.', 'newfold-performance-module' ),
+				'test'  => function () {
+					return isset( $_SERVER['HTTP_CF_RAY'] );
+				},
+			)
+		);
+
+		/* // phpcs:ignore Squiz.PHP.CommentedOutCode
+		// Enable when https://github.com/newfold-labs/wp-module-performance/pull/32 is merged.
+		// PRESS7-119: Lazy Loading.
+		$manager->addHealthCheck(
+			array(
+				'id'      => 'lazy-loading',
+				'title'   => __( 'Lazy Loading', 'newfold-performance-module' ),
+				'pass'    => __( 'Lazy loading is enabled', 'newfold-performance-module' ),
+				'fail'    => __( 'Lazy loading is disabled', 'newfold-performance-module' ),
+				'text'    => __( 'Lazy loading can improve performance by only loading images when they are in view.', 'newfold-performance-module' ),
+				'actions' => array(
+					array(
+						'label'    => __( 'Configure lazy loading.', 'newfold-performance-module' ),
+						'url'      => admin_url( 'admin.php?page=' . $this->container->plugin()->id . '&nfd-target=lazy-loading-enabled#/performance' ),
+						'external' => false,
+					),
+				),
+				'test'  => function () {
+					$enabled = get_option( 'nfd_image_optimization', array() );
+					return ( isset( $enabled['lazy_loading'], $enabled['lazy_loading']['enabled'] ) && $enabled['lazy_loading']['enabled'] );
+				},
+			)
+		); */
+
+		/* // phpcs:ignore Squiz.PHP.CommentedOutCode
+		// Enable when https://github.com/newfold-labs/wp-module-performance/pull/26 is merged.
+		// PRESS7-120: Link Prefetching.
```

Comment:
The Link Prefetch and Jetpack Boost stuff has been merged. I'm working on getting the Image Optimization changes merged as soon as possible after a review! 😃